coordinator/src/node/unrealized_pnl.rs

```diff
@@ -0,0 +1,73 @@
+use crate::db;
+use crate::node::Node;
+use crate::position::models::Position;
+use anyhow::Context;
+use anyhow::Result;
+use diesel::r2d2::ConnectionManager;
+use diesel::r2d2::PooledConnection;
+use diesel::PgConnection;
+use rust_decimal::Decimal;
+use time::OffsetDateTime;
+use trade::bitmex_client::BitmexClient;
+use trade::bitmex_client::Quote;
+use trade::cfd::calculate_pnl;
+use trade::Direction;
+
+pub async fn sync(node: Node) -> Result<()> {
+    let mut conn = node.pool.get()?;
+
+    let positions = db::positions::Position::get_all_open_or_closing_positions(&mut conn)?;
+    let current_quote = BitmexClient::get_quote(&OffsetDateTime::now_utc())
+        .await
+        .context("Failed to fetch quote from BitMEX")?;
+
+    for position in positions.iter() {
+        if let Err(e) = sync_position(&mut conn, position, current_quote.clone()) {
+            tracing::error!(position_id=%position.id, ?current_quote, "Failed to update position's unrealized pnl in database: {e:#}")
+        }
+    }
+
+    Ok(())
+}
+
+fn sync_position(
```

Comment:
🔧 Given my comments below, I think one or two tests would be really helpful, although better if they don't involve the DB at all. Maybe we could move the pure function that calculates the coordinator's PNL based on the (trader's) `Position` to `trade::cfd`?

Reply:
see https://github.com/get10101/10101/pull/967#discussion_r1271774521

I think it's better to keep the `cfd` module unaware of the "coordinator". 
I hope the tests are not too confusing - I was confused for a bit until I went back to our famous CFD hedging sheet to check the numbers...